src/components/body/Body.module.css

```diff
@@ -0,0 +1,6 @@
+.body {
+  display: flex;
+  flex-direction: row;
+  height: 95%;
```

Comment:
Height of the Header = 5%.
So, we should explicitly set the height of the body to 95%. 
Reason: In case we change the height of the header, the body also needs to be changed.

As the App is a flex container, we can simply grow the body to take up the available space.